Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -122,6 +122,18 @@ public class NetworkManager : MonoBehaviour
         //    during FinishLoadScene.
         public NetworkManagerMode mode { get; private set; }
 
+        [RuntimeInitializeOnLoadMethod]
+        static void Init()
```

Comment:
nitpick: Init() implies that something is initialized.
but in this case, it pretty much only resets state.
maybe call the methods Reset()?
that's more obvious, more consistent because we already have a few reset methods.
and Shutdown() could reuse them

Reply:
This was removed in favor of Shutdown, which was renamed to ResetStatics per discord convo